.github/workflows/add-discourse-topic.yml

```diff
@@ -0,0 +1,30 @@
+name: Add discourse topic
+on:
+  workflow_dispatch:
+jobs:
+  add-discourse-topic:
+    runs-on: ubuntu-latest
+    steps:
+
+      # Get the branch
+      - name: Get branch
```

Comment:
What branch does it check out? Is the GitHub Action meant to be used with feature branches?

Reply:
It checks out the branch that it was run upon.  Yes, I think we'd run it to make the topics right before merging a feature branch, but it's not 100% clear yet.  After we use it for a while, it'll probably be apparent where exactly it fits.